ros/src/util/packages/autoware_launcher/README.md

```diff
@@ -1,82 +1,17 @@
 # Autoware Launcher
 
-## Quick Start
-1. Put map and rosbag file
-```
-~/.autoware/data/tf/tf.launch
-~/.autoware/data/pointcloud_map/*.pcd
-~/.autoware/data/vector_map/*.csv
-~/.autoware/log/20150324.bag
-```
-1. Start the Autoware Launcher<br>
-```
-$ cd Autoware/ros
-$ ./run-experimental
-```
-![quickstart01](./documents/images/quickstart01.png)
-1. Load a profile if needed<br>
-Window Menu -> File -> Load Profile
-1. Show simulation panel<br>
-Window Menu -> View -> Simulation
-1. Play rosbag<br>
-Swtich on Simulation Mode check box, then, push Play button.
-1. Push launch buttons<br>
-Map, Vehicle, Sensing, and Visualization
-![quickstart02](./documents/images/quickstart02.png)
-1. Push Localization button on rviz plugin and check the estimated vehicle pose
-![quickstart03](./documents/images/quickstart03.png)
-1. Push other buttons on rviz plugin<br>
-Detection, Prediction, Decision, Mission, and Motion
-![quickstart04](./documents/images/quickstart04.png)
-![quickstart05](./documents/images/quickstart05.png)
+Autoware Launcher is a tool to launch Autoware easily. Developers can save the nodes and their parameters as profile. Users can use same configuration by loading profile.
```

Comment:
Autoware Launcher is designed to help users and developers to launch sets of nodes and its parameters. The profiles are stored in files that can be shared with other users, or used to quickly launch different set-ups in a breeze.

Reply:
Applied your suggestion.